### PR TITLE
Some INI related fixes

### PR DESCRIPTION
**Pull request by jonwil**

### CODE REVIEW

File: src/game/common/audio/audiomisc.cpp [hunk 1]
```diff
@@ -66,6 +66,7 @@ FieldParse MiscAudio::s_fieldParseTable[] = { { "RadarNotifyUnitUnderAttackSound
     { "AircraftWheelScreech", &INI::Parse_Audio_Event_RTS, nullptr, offsetof(MiscAudio, m_aircraftWheelScreech) },
     { nullptr, nullptr, nullptr, 0 } };
 
+// Was originally INI::parseMiscAudio
```
Comment (duncanspumpkin) [inline src/game/common/audio/audiomisc.cpp:69]: Why no definition for this one

Comment (jonwil) [reply src/game/common/audio/audiomisc.cpp:69]: I don't know, ask EA :)
The original function in the game is named INI::parseMiscAudio.

File: src/game/common/globaldata.cpp [hunk 2]
```diff
@@ -699,13 +699,6 @@ GlobalData::GlobalData()
     m_hardSoloAIHealthBonus = 1.0f;
```
Comment (tomsons26) [inline src/game/common/globaldata.cpp:699]: this made me notice the health bonuses were arrays

**Review by tomsons26 — CHANGES_REQUESTED** [review]

File: src/game/common/globaldata.cpp (hunk 2 above)
Comment (duncanspumpkin) [reply src/game/common/globaldata.cpp:699]: Are they? The INI parse function doesn't look to be using it as an array L319.

**Review by tomsons26 — APPROVED** [review]

File: src/game/common/globaldata.cpp (hunk 2 above)
Comment (tomsons26) [reply src/game/common/globaldata.cpp:699]: They are, ill pr the fix later

**Pull request merged**